server/src/main/java/org/apache/druid/server/coordinator/duty/KillUnusedSegments.java

```diff
@@ -249,6 +292,10 @@ private Interval findIntervalForKill(
     stats.add(Stats.Kill.ELIGIBLE_UNUSED_SEGMENTS, datasourceKey, unusedSegmentIntervals.size());
 
     if (CollectionUtils.isNullOrEmpty(unusedSegmentIntervals)) {
+      log.info(
```

Comment:
I think this will log for every datasource where we do not find any segments to kill. Do you think this might get too chatty?

Reply:
Good call, this is not really an exit condition, so I went ahead and removed this log. Thanks!